cdc/processor/processor.go

```diff
@@ -507,13 +508,13 @@ func (p *processor) Tick(ctx cdcContext.Context) error {
 	p.metricProcessorTickDuration.Observe(costTime.Seconds())
 	p.refreshMetrics()
 
-	if err == nil {
-		return nil
-	}
 	return p.handleErr(err)
 }
 
 func (p *processor) handleErr(err error) error {
+	if err == nil {
```

Comment:
Why do we move this inside?

Reply:
I just think it is more elegant.

Reply:
it looks that all the unit test in the file `processor_test.go` does not return an error from `p.Tick` method.

If it's possible, a new unit test covering return error logic is expected.